File: FrontEnd/src/app/features/sales/ledgers/classes/view-models/list/ledger-vm.ts
import { LedgerDocumentTypeVM } from './ledger-documentType-vm'
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface LedgerVM {

    date: string
    formattedDate: string
    shipOwner: SimpleEntity
    customer: SimpleEntity
    documentType: LedgerDocumentTypeVM
    invoiceNo: string
    debit: number
    credit: number
    balance: number

}

File: API/Features/Sales/Ledgers/ViewModels/Report/LedgerVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Sales.Ledgers {

    public class LedgerVM {

        public string Date { get; set; }
        public SimpleEntity ShipOwner { get; set; }
        public SimpleEntity Customer { get; set; }
        public DocumentTypeVM DocumentType { get; set; }
        public string InvoiceNo { get; set; }
        public decimal Debit { get; set; }
        public decimal Credit { get; set; }
        public decimal Balance { get; set; }

    }

}